lib/src/layer/marker_layer.dart

```diff
@@ -81,11 +81,15 @@ class Marker {
   final double height;
   final Anchor anchor;
 
+  /// If true marker will be counter rotated to the map rotation
+  final bool rotateMarker;
+
   Marker({
     this.point,
     this.builder,
     this.width = 30.0,
     this.height = 30.0,
+    this.rotateMarker = false,
```

Comment:
To keep it simple I would prefer to have the same name in the Marker and MakerLayerOptions.

This would be the default in MakerLayerOptions
rotate = false
rotateOrigin = null
rotateAlignment = Alignment.center

and this in Marker
rotate = null
rotateOrigin = null
rotateAlignment = null

Where each Marker option if not null overrides MakerLayerOptions.

Reply:
And in a follow up PR we maybe should replace AnchorAlign with Alignment so that both options use the same values.

Reply:
> To keep it simple I would prefer to have the same name in the Marker and MakerLayerOptions.
> 
> This would be the default in MakerLayerOptions
> rotate = false
> rotateOrigin = null
> rotateAlignment = Alignment.center
> 
> and this in Marker
> rotate = null
> rotateOrigin = null
> rotateAlignment = null
> 
> Where each Marker option if not null overrides MakerLayerOptions.

Yes, that could work, using `marker.rotate ?? markerOps.rotate`

Reply:
already updated it :)